pkg/bootstrap/bootstrap.go

```diff
@@ -161,23 +160,25 @@ func extractFromDir(dir, prefix string, img v1.Image, imgName string) error {
 		return err
 	}
 	defer os.RemoveAll(tempDir)
+
 	r := mutate.Extract(img)
 	defer r.Close()
 
 	// extracting manifests
 	if err := extract(imgName, tempDir, prefix, r); err != nil {
 		return err
 	}
-	if err := os.Rename(tempDir, dir); err != nil && err != os.ErrExist {
-		return err
-	} else if err == nil {
+
+	if err := os.Rename(tempDir, dir); err == nil {
```

Comment:
The old logic would return an error if it was not nil and wasnt an ErrExist. Just reasoning it out in my head, that makes sense. Can you explain how this change to remove that bit of error handling fixes the problem?

Reply:
The error returned is a LinkError and carries the underlying syscall error in it so it wouldn't actually match on ErrExists. This change simplifies for the expected scneario where the new path exists and continues silently. The other fail scneairos are os/fs related and would be evident from system state/stability.